src/schema/sale_artwork.js

```diff
@@ -166,6 +192,29 @@ const SaleArtworkType = new GraphQLObjectType({
             highest_bid
           ),
       },
+      increments: {
+        type: new GraphQLList(BidIncrementsFormatted),
+        resolve: async (
+          { minimum_next_bid_cents, sale_id, symbol },
+          options,
+          request,
+          { rootValue: { incrementsLoader, saleLoader } }
+        ) => {
+          return bid_increments_calculator({
+            sale_id,
+            saleLoader,
+            incrementsLoader,
+            minimum_next_bid_cents,
+          }).then(bid_increments => {
+            return bid_increments.map(increment => {
+              return {
+                cents: increment,
+                display: formatMoney(increment / 100, { symbol, precision: 0 }),
```

Comment:
#minor note, when we do an auction in JPY the `/100` won't be correct (but that can be solved by adding `subunit_to_unit` or whatever the appropriate ratio is to use). I'm sure there'll be...a few places we have to do that.

Reply:
Yeah definitely. I'm not looking forward to updating Eigen, and Force, and Causality (?), maybe prediction, Refraction, Eidolon... aw jeez.

Reply:
Yeah we don't handle this well anywhere yet 😫